vmdb/app/models/orchestration_template.rb

```diff
@@ -48,6 +48,11 @@ def parameter_groups
     raise NotImplementedError, "parameter_groups must be implemented in subclass"
   end
 
+  # List managers that may be able to deploy this template
+  def eligible_managers
```

Comment:
I think this should be a class method too.  I guess you could have an instance method that just calls the class_method if necessary.